src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/DefaultRazorDynamicFileInfoProvider.cs

```diff
@@ -272,11 +271,30 @@ public override bool Equals(object obj)
 
             public override int GetHashCode()
             {
-                var hash = new HashCodeCombiner();
-                hash.Add(ProjectFilePath, FilePathComparer.Instance);
-                hash.Add(FilePath, FilePathComparer.Instance);
+                var hash = 17;
```

Comment:
Just for my knowledge, how are these values (`17`, `23`) decided for generating the hash?

Reply:
Prime numbers make for fewer hashing collisions 😄

Reply:
Ah got it, thanks. So the numbers themselves are arbitrary (aside from being prime).

Reply:
I'm confused. Did we lose access to `HashCodeCombiner`?

Reply:
Thanks for calling me out @ajaybhargavb. I could have swore I changed this for a good reason but after trying to remove it things work as expected. Undoing